docs/source/docker/advanced/active_learning.rst

```diff
@@ -194,10 +194,10 @@ Run Active Learning using the Docker
 At this point you should have an `embeddings.csv` file with the active learning 
 scores in a column named `al_scores`. 
 
-We can now perform an active learning sampling using the `CORAL` sampler. In order
-to do the sampling on the `embeddings.csv` file we need to make this file
+We can now perform an active learning iteration using the `coral` selection strategy. In order
```

Comment:
we said CORAL, no?

Reply:
https://lightly-ai.slack.com/archives/C014FJT529M/p1649231912686499